ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-13.yaml

```diff
@@ -41,57 +52,82 @@ tests:
     - ref: stackrox-e2e
     workflow: ipi-aws
   timeout: 5h0m0s
-- as: qa-e2e-tests
+- as: ui-e2e-tests
   optional: true
-  run_if_changed: ^((generated|image|operator|pkg|make|deploy|scripts)/.*|Makefile|status\.sh|COLLECTOR_VERSION|EXPECTED_GO_VERSION)
+  run_if_changed: ^ui/
   steps:
-    cluster_profile: aws
+    cluster_profile: gcp
     env:
-      TEST_SUITE: ocp-qa-e2e-tests
+      TEST_SUITE: ocp-ui-e2e-tests
     test:
     - ref: stackrox-e2e
-    workflow: ipi-aws
+    workflow: ipi-gcp
   timeout: 5h0m0s
-- as: ui-e2e-tests
+- as: fips-qa-e2e-tests
```

Comment:
[Paying back with the same coin :-P ] Does it make sense to group `*qa-e2e-tests` next to each other?

Reply:
I would say Yes :) But does that fall on this PR? I used the same order in the original 4.12 to replace the previous order in 4.13.